iceoryx_examples/iceperf/iceoryx_wait.cpp

```diff
@@ -0,0 +1,52 @@
+// Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Please also add your copyright at the top of the file

Reply:
I hereby resign from my rights to the code.

Reply:
Okay, in this case we have to check whether a generic `Copyright 2023, Eclipse Foundation and the iceoryx contributors. All rights reserved.` would be sufficient. We did not yet have the situation of new files and no copyright holder.

Btw, you need to sign the ECA else we cannot merge the PR.

Fun fact, according to the law you cannot really resign from copyright rights. You will always be the copyright holder.

cc @budrus @mossmaurice

Reply:
@anderay it seems to be okay to resign from the copyright but you need to sign the ECA. This is required by the Eclipse Foundation and we cannot circumvent this since it is also part of the CI check.